assets/components/src/popover/style.scss

```diff
@@ -7,13 +7,13 @@
 
 .newspack-popover {
 	.components-popover__content {
-		border: 1px solid $light-gray-500;
+		border: 1px solid $gray-300;
```

Comment:
Would be nice to add an example of an action card with a popover to the components demo page.

Reply:
Might be worth re-working the ActionCard to allow this kind of pattern since we are using it in multiple places now (Campaigns, Ad Units, ...)